gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -1430,3 +1430,50 @@ nnstreamer_version_fetch (guint * major, guint * minor, guint * micro)
   if (micro)
     *micro = (NNSTREAMER_VERSION_MICRO);
 }
+
+/**
+ * @brief Compare dimension strings
+ * @return TRUE if equal, FALSE if given dimension strings are invalid or not equal.
+ */
+gboolean
+gst_tensors_dimension_string_equal (const gchar * dimstr1,
+    const gchar * dimstr2)
+{
```

Comment:
Is it the same function as using  g_strcmp0(https://docs.gtk.org/glib/func.strcmp0.html) ?
int 
g_strcmp0 (
  const char* str1,
  const char* str2
)

Reply:
Sorry.
I couldn't think of an order.